Source/Core/CommandLineOptions.cs

```diff
@@ -764,18 +764,13 @@ public enum TypeEncoding {
       }
     }
 
-    public IEnumerable<string/*!*/> ProcsToCheck {
-      get {
-        Contract.Ensures(cce.NonNullElements(Contract.Result<IEnumerable<string/*!*/>>(), true));
-        return this.procsToCheck != null ? this.procsToCheck.AsEnumerable() : null;
-      }
-    }
-
-    private List<string/*!*/> procsToCheck = null;  // null means "no restriction"
```

Comment:
Do not delete the comment.

Also, I have a slight preference for using null to represent "no restriction".  But it is not a big deal.

Reply:
I found this better (and more uniform with `procsToIgnore`) because it safes on non-null checks in several places.

It just represents what was given on the command line, a possibly empty list of `/proc`.

Reply:
Added back a comment.